ui/apps/platform/cypress/integration/administration/events/AdministrationEvents.helpers.js

```diff
@@ -0,0 +1,57 @@
+import { interactAndWaitForResponses } from '../../../helpers/request';
+import { visit } from '../../../helpers/visit';
+
+// Source of truth for keys in routeMatcherMap and staticResponseMap objects.
+export const eventAlias = 'administration/events/id';
+export const eventsAlias = 'administration/events';
+export const eventsCountAlias = 'count/administration/events';
```

Comment:
Yeah, apparently this is newest pattern from core team, because it was an explicit change a few weeks ago from the conventional eventscount path segment in earlier drafts.